.changeset/young-steaks-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+fix: optimize coin gql queries
```

Comment:
```suggestion
---
"@fuel-ts/account": minor
---

fix!: optimize coin gql queries
```

This will be a breaking change because we are removing the `getCoin` query, which you'll know longer be able to call on `Provider.operations.getCoin()`. This is okay with me, we can just recommend `Provider.operations.getCoins()` in breaking change notes.

Reply:
For speed, I'm pushing this change and will add the breaking change notes.